setup.cfg

```diff
@@ -1,2 +1,95 @@
+#
+# setup.cfg file for cleancut/green
+#
+# https://setuptools.pypa.io/en/latest/userguide/declarative_config.html
+#
+
 [metadata]
-description_file = README.md
+name = green
+summary = Green is a clean, colorful, fast python test runner.
+author = Nathan Stocks
+maintainer = Nathan Stocks
+author_email = nathan.stocks@gmail.com
+license = 'MIT'
+version = file: green/VERSION
+
+
+long_description = file:README-pypi.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+description_file = file:README.md
+description_content_type = text/markdown; charset=UTF-8
+
+url = https://github.com/CleanCut/green
+
+classifier =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Operating System :: MacOS :: MacOS X
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Operating System :: POSIX :: Linux
+    Operating System :: Unix
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3 :: Only
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: Quality Assurance
+    Topic :: Software Development :: Testing
+    Topic :: Utilities
+
+keywords =
+    nose, nose2, trial, pytest, py.test, tox, green,
+    tdd, test, tests, functional, system, unit, unittest,
+    color, tabular, clean, red, rednose, regression, runner,
+    integration,smoke, white, black, box, incremental, end,
+    end-to-end, sanity, acceptance, load, stress, performance,
+    usability, install, uninstall, recovery, security,
+    comparison, alpha, beta, non-functional, destructive,
+    accessibility, internationalization, i18n, localization, l10n,
+    development, a/b, concurrent, conformance, verification,
+    validation, quality, assurance, ad-hoc, agile, api,
+    automated, all, pairs, pairwise, boundary, value, branch,
+    browser, condition, coverage, dynamic, exploratory,
+    equivalence, partitioning, fuzz, gui, glass, gorilla,
+    interface, keyword, penetration, retesting, risk, based,
+    scalability, soak, volume, vulnerability
+
+
+
+test_suite='green.test'
+
+# TODO: find a way to get $version to work here.
+# download_url = https://github.com/CleanCut/green/tarball/$version
+
```

Comment:
From my research, I believe `download_url` is no longer used, so this can just be deleted.